docs/internal/internal-user-interface.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Internal User Interface
+---
+
+# Internal User Interface
+
+Our internal dashboard is primarily ERB views that render on the server.
+Largely, we try to adhere to
+[ActionView](https://guides.rubyonrails.org/action_view_overview.html)'s
+conventions in these views.
+
+For layout, basic styles, and some interactions, we use
+[Bootstrap 4](https://getbootstrap.com/). DEV isn't dedicated to using Bootstrap
+for everything, but because our design team hasn't spent much time on these
+views, we find it's easier to stick with something many developers already know.
+
+When a view requires some custom interactivity, we've historically leaned on
+vanilla JavaScript or jQuery, but going forward we've elected to use
+[StimulusJS](https://stimulusjs.org/) for DOM manipulation and interactivity
```

Comment:
Casually encouraging you to write a post about our use of Stimulus at some point :D